generators/GeneratorUtils/default_template.jl

```diff
@@ -0,0 +1,38 @@
+# Template showing the use of GeneratorUtils, with default I/O stdin and stdout. The output
+# is fine without modification on some exercises. Use the override! mechanism to control
+# output (see docs in GeneratorUtils).
+
+# This script calls 'using GeneratorUtils', so make sure GeneratorUtils.jl is available from
+# LOAD_PATH. For example, assuming the file is at GeneratorUtils/GeneratorUtils.jl, either
+# call julia with the JULIA_LOAD_PATH environment variable set, like so (note the ':', which
+# is important to make sure the path appends to rather than overwrites LOAD_PATH):
+#   JULIA_LOAD_PATH=:GeneratorUtils/ julia [this file] < canonical_data.json > runtests.jl
+# Or, within an interactive session:
+#   push!(LOAD_PATH, "GeneratorUtils/")
+
+using GeneratorUtils
+
+using DataStructures: OrderedDict
+using JSON
+
+# Modify as desired.
+const IO_IN = stdin
+const IO_OUT = stdout
```

Comment:
Nothing. I thought they looked better, but in my usual I/O workflow in notebooks I actually have global mutables `io_in = [...]` and `io_out = [...]` that I change to whatever is most convenient at the time. Would you recommend that, or do you mean you would go further to remove `io_in` and `io_out` and directly put in `stdin` and `stdout` where appropriate?

Reply:
If you find it useful then by all means leave it in. I can see the purpose for notebooks where you don't have a proper stdin or stdout, but I don't personally see why I would be editing those otherwise, but I don't see everything :)

Reply:
Okay, let's leave as-is for now.